src/Core/SettingsChangesHistory.cpp

```diff
@@ -58,6 +58,8 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"distributed_cache_read_request_max_tries", 20, 10, "Changed setting value"},
             {"distributed_cache_connect_max_tries", 20, 5, "Changed setting value"},
             {"opentelemetry_trace_cpu_scheduling", false, false, "New setting to trace `cpu_slot_preemption` feature."},
+            {"parallel_replicas_support_projection", false, true, "New setting. Optimization of projections can be applied in parallel replicas. Effective only with enabled parallel_replicas_local_plan and aggregation_in_order is inactive."},
+            {"vector_search_with_rescoring", true, true, "New setting."},
```

Comment:
`vector_search_with_rescoring` is now duplicated

Reply:
Sorry, I missed it when pulling from the master branch.